scripts/ci-entrypoint.sh

```diff
@@ -42,8 +42,9 @@ source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 # shellcheck source=hack/util.sh
 source "${REPO_ROOT}/hack/util.sh"
 
+# all test regions must support AvailabilityZones
 get_random_region() {
-    local REGIONS=("eastus" "eastus2" "northcentralus" "northeurope" "uksouth" "westeurope" "westus2")
```

Comment:
the one we want to change is actually ci-e2e.sh (that's the script for running the CAPZ e2e suite) but for the sake of consistency, let's remove every occurrence (there should be 3 total, you can search for "get_random_region" in the codebase to find them).

Reply:
Should be looking good now - thanks!